src/custom/components/AccountDetails/index.tsx

```diff
@@ -173,6 +173,7 @@ export default function AccountDetails({
 
   const handleDisconnectClick = () => {
     ;(connector as any).close()
+    localStorage.removeItem(STORAGE_KEY_LAST_PROVIDER)
```

Comment:
Is this the only way users might disconnect? 

should this be in a more central place? (like src/hooks/web3.ts , Web3ReactManager, or sth like that?)

Reply:
I'm not sure about this but for now I would add this.